Python/generated_cases.c.h

```diff
@@ -9268,34 +9268,29 @@
             frame->instr_ptr = next_instr;
             next_instr += 1;
             INSTRUCTION_STATS(LOAD_SPECIAL);
-            _PyStackRef owner;
-            _PyStackRef attr;
-            _PyStackRef self_or_null;
-            owner = stack_pointer[-1];
-            assert(oparg <= SPECIAL_MAX);
-            PyObject *owner_o = PyStackRef_AsPyObjectSteal(owner);
+            _PyStackRef self;
+            _PyStackRef *method_and_self;
+            self = stack_pointer[-1];
+            method_and_self = &stack_pointer[-1];
+            method_and_self[1] = self;
+            method_and_self[0] = PyStackRef_NULL;
             PyObject *name = _Py_SpecialMethods[oparg].name;
-            PyObject *self_or_null_o;
             stack_pointer += -1;
             assert(WITHIN_STACK_BOUNDS());
             _PyFrame_SetStackPointer(frame, stack_pointer);
-            PyObject *attr_o = _PyObject_LookupSpecialMethod(owner_o, name, &self_or_null_o);
+            int err = _PyObject_LookupSpecialMethod(name, method_and_self);
```

Comment:
@markshannon  - I switched to the simpler formulation, but the stackpointer adjustments aren't safe here for the free threaded build. 

The `stack_pointer += -1;` means that `self` is not visible on the stack anymore when `_PyObject_LookupSpecialMethod` is called. In other words the code is generated as if the inputs are dead (okay), but the outputs are not yet alive (bad).